core/src/main/java/org/stellar/anchor/sep12/Sep12Service.java

```diff
@@ -68,20 +68,24 @@ public void deleteCustomer(JwtToken jwtToken, String account, String memo, Strin
           String.format("Not authorized to delete account [%s]", account));
     }
 
-    Sep12GetCustomerResponse existingCustomer =
-        customerIntegration.getCustomer(
-            Sep12GetCustomerRequest.builder()
-                .account(account)
-                .memo(memo)
-                .memoType(memoType)
-                .build());
-    if (existingCustomer.getId() == null) {
-      infoF(
-          "No existing customer found for account={} memo={} memoType={}", account, memo, memoType);
-      throw new SepNotFoundException("User not found.");
-    }
+    String senderTypes[] = {"sep31-sender", "sep31-receiver"};
+    for (String senderType: senderTypes) {
+      Sep12GetCustomerResponse existingCustomer =
+          customerIntegration.getCustomer(
+              Sep12GetCustomerRequest.builder()
+                  .account(account)
+                  .memo(memo)
+                  .memoType(memoType)
+                  .type(senderType)
+                  .build());
+      if (existingCustomer.getId() == null) {
```

Comment:
!!  You should only throw a 404 not found if none of the requests in the for loop succeed.

Reply:
Good catch, I'll alter the logic to only error if no match is found.